src/repl-window.ts

```diff
@@ -198,17 +199,23 @@ class REPLWindow {
 let ctx: vscode.ExtensionContext
 
 let replWindows: { [id: string]: REPLWindow } = {};
-let replViewColum: { [id: string]: vscode.ViewColumn } = {"clj": vscode.ViewColumn.Two, 
-                                                          "cljs": vscode.ViewColumn.Two};
+let replViewColum: { [id: string]: vscode.ViewColumn } = {
+    "clj": vscode.ViewColumn.Two,
+    "cljs": vscode.ViewColumn.Two
+};
+
+export function getReplWindow(mode: "clj" | "cljs") {
+    return replWindows[mode];
+}
 
 function getImageUrl(name: string) {
     let imagepath = "";
     if (!name)
-         imagepath = path.join(ctx.extensionPath, "assets/images/empty.svg");
+        imagepath = path.join(ctx.extensionPath, "assets/images/empty.svg");
     else
-         imagepath = path.join(ctx.extensionPath, "assets/images/", name);
+        imagepath = path.join(ctx.extensionPath, "assets/images/", name);
```

Comment:
i think we should add here `{ }` 
it would be more consistent with the rest of the code

Reply:
Agree.

This PR doesn't care about imageUrls at all, but Kaizen and all that. Update coming.

 We should have that convention documented somewhere, btw.

Reply:
Yes, we should add a "Convention" Section to the "How to Contribute" :)